static/bidder-info/ccx.yaml

```diff
@@ -0,0 +1,13 @@
+endpoint: "https://delivery.clickonometrics.pl/ortb/prebid/bid"
+maintainer:
+  email: "it@clickonometrics.pl"
+gvlVendorID: 773
+capabilities:
+  site:
+    mediaTypes:
+      - banner
```

Comment:
I'm noticing a discrepancy between the supported media types in this file vs. what's present in your dev docs. Here it says banner is the media type supported, but the doc says video. Could you address this?

Reply:
added

Reply:
Thanks for updating your .yaml file, but could you also update your docs PR to include the banner media type as well?

Reply:
@AlexBVolcy I see [PR #4127](https://github.com/prebid/prebid.github.io/pull/4127) was opened to address this.